### PR TITLE
Update ubuntu article

### PR DESCRIPTION
**Pull request by ahoneybun**
Addresses issue https://github.com/system76/docs/issues/1129

Updated the following:

- Removed System76 PPA preference file and nomodeset sections
- Added Disclaimer about non-LTS releases 
- Updated Slideshow screenshot

### CODE REVIEW

**Comment by leviport** [comment]
>Addresses issue https://github.com/system76/docs/issues/1129

I can't verify this since #1129 is an empty issue :stuck_out_tongue:

File: content/install-ubuntu.md [hunk 1]
```diff
@@ -124,7 +125,7 @@ This is when you will create your first user in your new install of Ubuntu.
 
 ### Slideshow
 
-![Slideshow](/images/install-ubuntu/install-ubuntu-21.04-7.png)
+![Slideshow](/images/install-ubuntu/install-ubuntu-22.04-7.png)
```
Comment (leviport) [inline content/install-ubuntu.md:128]: I see the new image added, but the old one was not removed. Should we get rid of the old one to save space?

**Comment by ahoneybun** [comment]
> > Addresses issue #1129
> 
> I can't verify this since #1129 is an empty issue stuck_out_tongue

Blame @thomas-zimmerman

**Review by leviport — APPROVED** [review]

**Review by thomas-zimmerman — APPROVED** [review]
let's get this live!

**Comment by leviport** [comment]
No fair, I get in trouble when I deploy on Fridays...

**Pull request merged**